app/src/main/res/layout/all_episodes_fragment.xml

```diff
@@ -19,14 +33,15 @@
 
     <ProgressBar
         android:id="@+id/progLoading"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_gravity="center"
+        android:layout_width="match_parent"
+        android:layout_height="64dp"
+        android:gravity="center"
         android:indeterminateOnly="true"
         android:visibility="gone"
+        android:layout_alignParentBottom="true"
```

Comment:
Adding `layout_centerInParent` does the trick

```suggestion
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:layout_centerInParent="true"
        android:indeterminateOnly="true"
```

Reply:
Done that change.
Layouts are still foreign to me :-)